ruby.rb

```diff
@@ -0,0 +1,19 @@
+class Hound
+
+  def hello
+    'hello'
+  end
+
+  def hash
+    {
+      :long_key => "foo",
+      :foo      => "bar"
+    }
+  end
+
+  def iteration
+    [1,2,3].collect do |i|
```

Comment:
Prefer `map` over `collect`.<br>Space missing after comma.<br>Pass `&:to_s` as an argument to `collect` instead of a block.